frontend/src/components/shared/AICopilot.css

```
@@ -0,0 +1,477 @@
+  border-radius: 4px;
+  opacity: 0.3;
+}
+
+/* Message bubbles */
+.copilot-msg {
+  max-width: 85%;
+  padding: 10px 14px;
+  border-radius: 16px;
+  font-size: 13px;
+  line-height: 1.55;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)